docs/source/contributing/local-deployment.rst

```diff
@@ -169,32 +163,28 @@ Launching
 
       skaffold run -p dev,arm64
 
+.. tip::
+   If you need to re-run `skaffold run` for whatever reason, don't forget to use `skaffold delete` to reset the state beforehand (or reset your environment by running the `k3-create.sh` script again).
```

Comment:
Possibly the second part is a bit superfluous, there are many ways to reset the environment so giving one might be enough.

Reply:
Sometimes the `skaffold delete` ends with errors, and remembering to use the `k3-create` script again has helped me as a total k8s newbie to set things straight again.